libs/langchain-google-common/src/utils/common.ts

```diff
@@ -62,32 +62,38 @@ function processToolChoice(
 }
 
 export function convertToGeminiTools(tools: GoogleAIToolType[]): GeminiTool[] {
-  const geminiTools: GeminiTool[] = [
-    {
-      functionDeclarations: [],
-    },
-  ];
+  const geminiTools: GeminiTool[] = [];
   tools.forEach((tool) => {
     if (
       "functionDeclarations" in tool &&
       Array.isArray(tool.functionDeclarations)
     ) {
       const funcs: GeminiFunctionDeclaration[] = tool.functionDeclarations;
-      geminiTools[0].functionDeclarations?.push(...funcs);
+      geminiTools.push({ functionDeclarations: [...funcs] });
```

Comment:
I vaguely remember this not working - can you add a test using multiple traditional tools?

Reply:
Hello @afirstenberg and @jacoblee93,

Thank you for the review! We’re glad to hear that you appreciate our code contribution. We’ll make sure to add the additional test cases as requested. Please note that libs/langchain-google-vertexai/src/tests/chat_models.int.test.ts already includes test cases for grounding.

Reply:
I think there are some known bugs on Google's side about having multiple tools at once - but Google considers this a bug on their end. It *should* be possible to submit multiple tools at once and mixing functions and other tools.

Reply:
Yes this breaks multiple functions, will revert partially